shard/src/api/api.py

```diff
@@ -73,8 +77,47 @@ async def set_response(self, user_id, guild_id, trigger, response):
 
     @fetch_guild
     async def get_playlist(self, guild):
-        voice = self.bot.cogs['Voice'].voice_managers[guild.id]
-        return voice.q.as_dict(), sc.OK_200
+        voice = self.bot.lavalink.player_manager.get(guild)
+        if voice is None:
+            return {}, sc.OK_200
+        else:
+            songs = []
+            for s in voice.queue:
+                songs.append({'title': s.title, 'author': s.author, 'duration': s.duration, 'uri': s.uri})
```

Comment:
what happened to `as_dict`

Reply:
That's a custom method you implemented on your custom queue class. The lavalink queue doesn't have it.

Reply:
I know, I'm saying add it instead of creating a new dictionary of unknown structure every 2 lines